#!/bin/sh /usr/share/dpatch/dpatch-run
# Description: Only install EFI stuff if EFI is actually enabled.
# Origin: upstream, http://www.virtualbox.org/changeset/29077

@DPATCH@
--- trunk/Makefile.kmk (revision 28800)
+++ trunk/Makefile.kmk (revision 29077)
@@ -321,4 +321,5 @@
 # Install EFI firmware image
 #
+ifdef VBOX_WITH_EFI
 ifdef VBOX_WITH_EFIFW_FROM_BUILD_SERVER
 #
@@ -374,4 +375,5 @@
         $(PATH_ROOT)/src/VBox/Devices/EFI/FirmwareBin/VBoxEFI64.fd=>VBoxEFI64.fd
 endif # VBOX_WITH_EFIFW_FROM_BUILD_SERVER
+endif # VBOX_WITH_EFI
 
 
